demisto_sdk/scripts/validate_mypy_global_ignore.py

```diff
@@ -0,0 +1,101 @@
+import re
+from pathlib import Path
+from typing import Any, Dict, List, Tuple, Union
+
+import typer
+
+from demisto_sdk.commands.common.constants import DEMISTO_GIT_PRIMARY_BRANCH
+from demisto_sdk.commands.common.git_util import GitUtil
+from demisto_sdk.commands.common.logger import logger, logging_setup
+
+MYPY_GLOBAL_IGNORE_PATTERN = re.compile(r"^#\s*type\s*:\s*ignore")
+
+main = typer.Typer()
+
+
+def has_global_type_ignore(file_path: Path) -> Tuple[bool, Union[int, None]]:
+    """
+    Helper function to check whether the input file
+    has the global mypy ignore set.
+
+    Args:
+    - `file_path_str` (``str``): The input Python file.
+
+    Returns:
+    - `True` if the file is globally ignored, `False`
+    otherwise.
+    - `int` with the line number where the ignore comment
+    was found, `None` if it wasn't found.
+    """
+
+    lines = file_path.read_text().splitlines()
+
+    for line_num, line in enumerate(lines):
+        if MYPY_GLOBAL_IGNORE_PATTERN.match(line):
+            return True, line_num + 1
```

Comment:
we can just return `line_num+`, or `None` if not found, no need for the boolean (simplifies the code)

Reply:
https://github.com/demisto/demisto-sdk/pull/4405/commits/7aa41713839fc79068afff7a3582c7a1fa9f824e